node/common/transport/amqp/lib/amqp_receiver.js

```diff
@@ -80,7 +82,7 @@ AmqpReceiver.prototype._removeAmqpReceiverListeners = function () {
 AmqpReceiver.prototype.complete = function (message, done) {
   if (!message) { throw new ReferenceError('Invalid message object.'); }
   this._amqpReceiver.accept(message);
-  if(done) done();
+  if(done) done(null);
```

Comment:
yeah that's an interesting one. basically, node callbacks (nodebacks) assume we should call the callback with (err, result) parameters. we also have somewhere in the requirement docs that err shall be null when successful. ultimately, it should probably be (null, SomeTransportAgnosticResult).